docs/developer/ceed.md

```diff
@@ -0,0 +1,206 @@
+
+# RDycore SWE physics with CEED
+
+Here we describe the RDycore's implemetation of 2D shallow water equation (SWE) that uses [libCEED](https://github.com/CEED/libCEED). The combination of PETSc and libCEED provides RDDycore algorithimic and hardware protability.
+
+- RDycore uses PETSc's `TS` solvers that provide support for multiple time-integrators such as forward euler, RK4, etc., which can be selected at run time (e.g. `--ts_type euler`, `--ts_type rk4`, etc). 
+- The use libCEED allows RDycore to compute the RHS-function for the explicit time-integrators on CPU or GPU, which can also be selected at runtime via
```

Comment:
The use libCEED -> Using libCEED

the RHS-function -> source and forcing terms (?)